src/applications/personalization/profile/tests/e2e/address-validation/addressValidation.js

```diff
@@ -0,0 +1,307 @@
+export const createAddressValidationResponse = type => {
+  if (type === 'low-confidence') {
+    return {
+      addresses: [
```

Comment:
Just because I'm being picky, I'd like to use a real response that has a confidence score in the 80s. This is for one reason in particular: it's be super easy for us to adjust the confidence score threshold we use to determine if we show the validation view or not. It used to be if the score was >80, we didn't show the screen. Then we decided to raise that threshold to 90. So I'd like this test to make sure we don't regress without catching it. It's actually easy to get a confidence score in the 80s. Just enter a valid address _but_ get the zip code wrong. Here is an example response:

```
{
  "addresses": [
    {
      "address": {
        "addressLine1": "36320 Coronado Dr",
        "addressType": "DOMESTIC",
        "city": "Fremont",
        "countryName": "United States",
        "countryCodeIso3": "USA",
        "countyCode": "06001",
        "countyName": "Alameda",
        "stateCode": "CA",
        "zipCode": "94536",
        "zipCodeSuffix": "5537"
      },
      "addressMetaData": {
        "confidenceScore": 81.0,
        "addressType": "Domestic",
        "deliveryPointValidation": "CONFIRMED",
        "residentialDeliveryIndicator": "RESIDENTIAL"
      }
    }
  ],
  "validationKey": -1808739503
}
```